cypress/e2e/GUI/orderingPagination.cy.js

```diff
@@ -0,0 +1,187 @@
+const firstClientRecord = 'table tbody tr:first-of-type button'
+const inputName = '[data-testid="name"]'
+const tableHeadNumberOfEmployees = 'tbody tr td:nth-child(3)'
+const tableHeadSize = 'thead > tr > :nth-child(4) > button'
+const firstTdTable = 'tbody tr:first-child td:nth-child(4)'
+const selectLimitDisplayRecords = 'select[aria-label="Pagination limit"]'
+
+describe('Ordenação e Paginacao', () => {
+
+
+    beforeEach(() => {
+        cy.visit('/')
+    });
+
+    it('Desativa o campo de entrada de texto quando na página de detalhes do cliente', () => {
+
+        cy.get(firstClientRecord, { timeout: 30000 }).should('be.visible').eq(0).click();
+        cy.get(inputName).should('be.visible').and('have.attr', 'disabled')
+
+    });
+
+
+
+    it('Verificar se é possivel ordenar por número de funcionários em ordem ascendente', () => {
```

Comment:
😱 
Quem testa que essa lógica toda funciona?

**Testes automatizados devem ser lineares. Pré-condição, ação e resultado esperado.**

Nada de `for` _loops_, condicionais, etc.